docs/how-to/how-to-comply-security.md

```diff
@@ -0,0 +1,23 @@
+# How to comply with security requirements
+
+[Running GitHub self-hosted runner poses security risk, such as, arbitrary code execution according to GitHub](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security). Like GitHub the self-hosted runners managed by the charm are isolated in its own single-use virtual machine instance. In addition, the charm enforce some repository settings to ensure all code running on the self-hosted runners are reviewed by someone trusted.
```

Comment:
s/Running GitHub self-hosted runner poses security risk, such as, arbitrary code execution according to GitHub/Running code inside the GitHub self-hosted runner poses significant security risk of arbitrary code execution according to GitHub